rotkehlchen/exchanges/binance.py

```diff
@@ -680,6 +681,73 @@ def _query_margined_futures_balances(
 
         return balances
 
+    def _query_pools_balances(
+            self,
+            balances: DefaultDict[Asset, Balance],
+    ) -> DefaultDict[Asset, Balance]:
+
+        def process_pool_asset(asset_name: str, asset_amount: FVal) -> None:
+
+            if asset_amount == ZERO:
+                return None
```

Comment:
unecessary whitespace
```suggestion
            if asset_amount == ZERO:
                return None
```